src/components/ShoppingCart/CartListItem.tsx

```diff
@@ -4,12 +4,17 @@ import { FaPen } from "react-icons/fa";
 import { MdDeleteForever } from "react-icons/md";
 import { ICartItem } from "@/interfaces/cartItem";
 
-const CartListItem = (props: ICartItem) => {
+interface CartListItemProps {
+  item: ICartItem;
+  onDelete: (id: string) => void;
+}
+
+const CartListItem = ({ item, onDelete }: CartListItemProps) => {
   // import cart information
-  const productName = props.design.designName;
-  const quotation = props.quotation;
-  const quotationDetails = props.quotationDetails;
-  const image = props.previewPic;
+  const productName = item.design.designName;
+  const quotation = item.quotation;
+  const quotationDetails = item.quotationDetails;
+  const image = item.previewPic;
```

Comment:
Do you know "destructuring assignment"? It can define several variables at once.

Reply:
We simply follows others style and actually we should use destructuring assignment.